### PR TITLE
Add support for Audio Profiles

### PR DESCRIPTION
**Pull request by gogasca**
Synthesize Text with Audio Profiles

### CODE REVIEW

File: texttospeech/src/synthesize_text_effects_profile.php [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * Copyright 2018 Google Inc.
```
Comment (bshaffer) [inline texttospeech/src/synthesize_text_effects_profile.php]: 2019

File: texttospeech/src/synthesize_text_effects_profile.php [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+use Google\Cloud\TextToSpeech\V1\SynthesisInput;
+use Google\Cloud\TextToSpeech\V1\TextToSpeechClient;
+use Google\Cloud\TextToSpeech\V1\VoiceSelectionParams;
+
+function synthesize_text_effects_profile($text, $effects_profile_id)
+{
+    // create client object
+    $client = new TextToSpeechClient();
+
+    $input_text = (new SynthesisInput())
```
Comment (bshaffer) [inline texttospeech/src/synthesize_text_effects_profile.php]: should be `$inputText`

File: texttospeech/src/synthesize_text_effects_profile.php [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+// [START tts_synthesize_text_audio_profile]
+use Google\Cloud\TextToSpeech\V1\AudioConfig;
+use Google\Cloud\TextToSpeech\V1\AudioEncoding;
+use Google\Cloud\TextToSpeech\V1\SsmlVoiceGender;
+use Google\Cloud\TextToSpeech\V1\SynthesisInput;
+use Google\Cloud\TextToSpeech\V1\TextToSpeechClient;
+use Google\Cloud\TextToSpeech\V1\VoiceSelectionParams;
+
+function synthesize_text_effects_profile($text, $effects_profile_id)
```
Comment (bshaffer) [inline texttospeech/src/synthesize_text_effects_profile.php]: should be `$effectsProfileId`

**Comment by gogasca** [comment]
Thanks for the review PTAL, changes submitted

**Pull request merged**